src/system_information/resources.rs

```diff
@@ -33,13 +35,16 @@ impl Resources {
         );
 
         let cpu_count = sys.cpus().len();
-        let physical_core_count = sys.physical_core_count();
+        let physical_core_count = System::physical_core_count();
         tracing::info!(
             cpus.physical = cpu_count,
             cpus.cores.physical = physical_core_count,
             "cpus"
         );
 
+        let open_files_limit = System::open_files_limit();
+        tracing::info!(open_files.limit = open_files_limit, "open files");
```

Comment:
I think this describes it better
```suggestion
        tracing::info!(open_files.limit = open_files_limit, "open file limit");
```

Reply:
[`485a391` (#45)](https://github.com/stackabletech/containerdebug/pull/45/commits/485a391181a91c7097dd2d693971139d5e94abca)